pages/common/httping.md

```diff
@@ -0,0 +1,19 @@
+# httping
+
+> Measure the latency and throughput of a webserver.
+
+- Ping the webserver on host 'localhost':
+
+`httping -g http://localhost/`
```

Comment:
Could we surround `http://localhost/` in `{{` and `}}` please?

Reply:
updated, thank you:)